### PR TITLE
25: Add documentation for building an Any Page

### PR DESCRIPTION
**Pull request by jimwood**
Demonstrate query/template pairing, web components and custom css

Relates to: https://jira.duraspace.org/browse/VIVO-1715

### CODE REVIEW

File: templates/element_doc_pages/any-page.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+<p>It will list people and documents related to a specified term illustrating:</p>
+<ul>
+  <li>Template and query pairing</li>
+  <li>Use of existing web components</li>
+  <li>Use of custom css</li>
+</ul>
+
+<ol>
+  <li>
+    <h3>Define a Template - <small><em>templates/any_pages/spotlight_on.html</em></small></h3>
```
Comment (awoods) [inline templates/element_doc_pages/any-page.html]: Is it correct to gather that the names of `whatever.html` has to match `whatever.graphql`?

Comment (jimwood) [reply templates/element_doc_pages/any-page.html]: Yes, they must match

Comment (awoods) [reply templates/element_doc_pages/any-page.html]: A comment to that affect my avoid future confusion.

Comment (jimwood) [reply templates/element_doc_pages/any-page.html]: Sounds good, I'll work one in. Thanks!

**Pull request merged**